features/editor_can_review_third-party_articles.feature

```diff
@@ -0,0 +1,21 @@
+@javascript 
+@api_call
+Feature: Display third-party articles
+    As an Editor,
+    In order to review external articles of interest
+    I would like to see a list of third party articles available.
+
+    Background:
+        Given I am on the third-party articles page
+
+    Scenario: View list of "News" category
+        Then I should see "News"
+        And I should see '.article-title' element
```

Comment:
I haven't used Webmock before - does this stub out the API call and present something different to test? If not, how are we testing that the given "News" articles are, in fact, in the "News" category?

Reply:
in the stub request, I am calling the same link I use to get the general/news category. I don't think we are really testing the specific articles with webmocks but more like faking it, eg. reduce api request(in case there are limits). In fact, when testing, it's not showing me new api response, but gives me the one i added in webmocks. If you want we can discuss about this

Reply:
sure - when you have a moment perhaps you can talk me through it as I miss this part of the presentation.